src/util/path.ts

```diff
@@ -8,8 +8,6 @@ import {
   PATH_DELIMITER
 } from "../common";
 
-const separator = CharCode.SLASH;
```

Comment:
Can we keep this? Idea was that we can simply make it mutable if dealing with Windows paths ever becomes a thing.

Reply:
Sure. Done